backend/spec/api/v3/groups_controller_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
+RSpec.describe Api::V3::GroupsController, type: :request do
+  describe 'api' do
+
+    # Sign in users
+    before :all do
+      @token_1 = []
+
+      post "/api/v3/token/create?login=user_1&password=password"
```

Comment:
I can do that.  It is always doing the same thing (i.e., always logging in one or more users, because a user can't do anything unless they are logged in...) so I thought it was pretty straightforward.  But I can see using a constant for "api/v3/token/create".

Reply:
Done. Will be in next push.